docs/examples/calling-external-api-lookup.md

```diff
@@ -0,0 +1,255 @@
+
+```js
+const response = await fetch(
+  `https://www.boredapi.com/api/activity?type=${activityType}`,
+  {
+    method: "GET",
+  }
+)
+```
+
+Now, we need to grab the data that returned from the call, we can do this by running response.json(), shown below:
+
+```js
+const activityReturned = await response.json();
+```
+
+The data that is returned from this API looks like this:
+```json
+{
+  "activity": "Learn how to play a new sport",
+  "type": "recreational",
+  "participants": 1,
+  "price": 0.1,
+  "link": "",
+  "key": "5808228",
+  "accessibility": 0.2
+}
+```
+
+We have now gotten our data from the API! Hooray! 
+But, what if we hadn't? Hooray... question mark?
+
+We can't assume that we will ALWAYS get a response of data, so we once again need error handling! This time we will just write:
+```js
+if (!activityReturned){
+    return res.setStatusCode(400).setPayload({
+        message: "Error, nothing returned from the API."
+    })
+}
+```
+
+This will ensure that if there is no data returned from the API, we will not face issues where a user is able to throw random errors due to somehow getting a null/undefined/unknown/etc. response from the API. We need to have a person for our next step or we will face issues! 
+
+Now that we have covered that, let us write the last part of the lookup!
+
+### Format and return that data from the lookup
+
+We have the data we require, we need to now return it for the user. We can do this by writing this following code:
+```js
+return res.setStatusCode(200).setPayload({
+  activity: activityReturned.activity,
+  participants: +activityReturned.participants,
+  price: +activityReturned.price,
+  link: activityReturned.link,
+  key: activityReturned.key,
+  accessibility: +activityReturned.accessibility,
+});
+```
+
+Now, we are returning the specific data in the payload for the user. We are returning the data in the specific spots we want them on the form, hence we are matching up what we grabbed from the API Object and placing as the value for specific form elements. Now we get to update the `.blinkmrc.json` file to complete this process. 
+
+### Update the `.blinkmrc.json`
+
+For us to use this new code we have written, we need to update the `.blinkmrc.json` file, so let us open it and add some new json to our `routes`. 
+You can copy below but we are going to add this:
+
+```json
+{
+  "route": "/data-from-external-api",
+  "module": "./src/fill-data-from-api.js"
+}
+```
+
+In this case, I stored my code in the folder path mentioned above. It's folder scheme would look something like this from the root folder of the API:
+
+```
+|-- project-root
+|   |-- .blinkmrc.json
+|   |-- src
+|   |   |-- routes
+|   |   |   |-- fill-data-from-api.js
+```
+
+## Using the new Code on the Console
+
+Now that we have written all of this code, we can now deploy it to the Console. If you have not hosted an API via OneBlink before, you can [follow this article here to understand how](../api/hosting-api.md).
```

Comment:
Dont think that link exists

Reply:
The link is to a file that is in the other MR, so when they are both merged, the link should work :) I initially wrote this MR while still in the other branch and separated them for when I wanted to create their MRs.

Reply:
This PR should probably target that other branch then?

Reply:
Targetting the other branch now.